tasks/1.test.js

```diff
@@ -0,0 +1,20 @@
+const { rangeArr, rangeWhile } = require("./1");
+
+describe("Return range to array", () => {
+  describe("For loop", () => {
+    test("Should return an array filled with all numbers from x (inclusive) to y (exclusive)", () => {
+      const result = rangeArr(1, 5);
+      const expected = [1, 2, 3, 4, 5];
+
+      expect(result).toStrictEqual(expected);
+    });
+  });
+  describe("While loop", () => {
+    test("Should return an array filled with all numbers from x (inclusive) to y (exclusive)", () => {
+      const result1 = rangeWhile(1, 5);
```

Comment:
Nitpick: variables names can repeat between tests as `const` and `let` are block scoped anyways. 😉  
Also, you could consider putting `expected` array in the top level `describe` as both of your tests are expecting the same result.